example/config.ts

```diff
@@ -0,0 +1,72 @@
+import { PathOptions, StyleFunction, WMSOptions } from 'leaflet'
+
+interface Example {
+  id: string
+  url: string
+  title: string
+}
+
+interface GeoJsonConfig<P = any> extends Example {
+  style?: PathOptions | StyleFunction<P>
+  activeStyle?: PathOptions
+}
+
+interface NonTiledConfig extends Example {
+  options: WMSOptions
+}
+
+const MAP_API_ROOT = 'https://map.data.amsterdam.nl/'
+
+export const GEO_JSON_LAYER_EXAMPLES: GeoJsonConfig[] = [
+  {
+    title: 'Parkeervlakken',
+    id: 'parkeerVlakken',
+    url: `${MAP_API_ROOT}maps/parkeervakken?REQUEST=Getfeature&VERSION=1.1.0&SERVICE=wfs&TYPENAME=alle_parkeervakken&srsName=EPSG:4326&Filter=%3CFilter%3E%3CPropertyIsEqualTo%3E%3CPropertyName%3Ee_type%3C/PropertyName%3E%3CLiteral%3EE6a%3C/Literal%3E%3C/PropertyIsEqualTo%3E%3C/Filter%3E&count=10000&startindex=0&outputformat=geojson`,
```

Comment:
it'a a bad practice to read 1000 polygons and draw in the map, a solution for these situations would be to limit the results by min and max zoomlevels combined with the current map bounding box

Reply:
Agreed. Although zoom-levels / clustering / passing a boundingbox is pretty implementation specific, I'd like to see if I can implement this in this example. Do you know how to pass the boundingbox to the filters? I have no idea to do that...

Reply:
@aditudorache I've added an issue for this. https://github.com/Amsterdam/amsterdam-react-maps/issues/21 I guess it's a pretty complex thing to just add to this prototype. The sole purpose of this PR was to set up a structure, tools and add some random examples. Therefore I'd consider it out of scope for now.

Reply:
Ok, I'll  create an example in other project and share it here!